### ISSUE TITLE
SCSS error after setting site to theme

### ISSUE THREAD

**Issue by emmarichardson** [gjb2048/moodle-theme_foundation#14]
<img width="880" alt="Screen Shot 2022-04-11 at 10 19 08 AM" src="https://user-images.githubusercontent.com/2079129/162785705-3895f0c4-19a4-4fac-a2e5-b95f0b71ac1d.png">
See screenshot.  Moodle 4.0 RC1

**Comment by gjb2048** [comment]
Thanks for the report @emmarichardson, what version of the theme and Moodle please?

**Comment by emmarichardson** [comment]
Put Moodle version above.  4.0 version of theme.

**Comment by gjb2048** [comment]
Thanks, sometimes the obvious is right in front of your nose.

**Comment by emmarichardson** [comment]
Actually I added that a couple of seconds after submitting so if you were really quick, you would have missed it!  Thanks for getting this out so fast by the way!

**Comment by emmarichardson** [comment]
I also noticed and just realized that it is not intentional and presuming it is related to scss file that all blocks are going to the bottom right of page..

**Comment by gjb2048** [comment]
@emmarichardson No worries, but cannot replicate.  Is there anything in your PHP log file?

Also, I've not updated the fallback sheet (has to be generated manually), so could be the reason for the blocks.  The sheet occurs when the SCSS fails so that you have a chance of a working'ish system for debugging purposes.

**Comment by gjb2048** [comment]
@emmarichardson Also, you can debug the SCSS compilation by altering the core file 'lib/classes/scss.php' like so:
![Screenshot 2022-04-11 174652](https://user-images.githubusercontent.com/1058419/162790439-eeac956e-bbb8-485a-8de9-20ad9c7bf77c.png)

I'm on Windows, so clearly change the filename / path as appropriate.

**Comment by emmarichardson** [comment]
Ok, so not sure what happened there.  Added that line (pretty sure I got the path to the file wrong - never quite sure how to do that when working in Mamp going to Mac) but it just started working!  Removed the line from the scss.php file and it is still working.  Maybe something about saving the file refreshed the cache?? So I guess I will go ahead and close this out.  Will let you know if any other oddities pop up.
